Source/WebCore/platform/graphics/GraphicsContextGL.h

```diff
@@ -1654,7 +1654,7 @@ class GraphicsContextGL : public RefCounted<GraphicsContextGL> {
 #if ENABLE(MEDIA_STREAM) || ENABLE(WEB_CODECS)
     virtual RefPtr<VideoFrame> surfaceBufferToVideoFrame(SurfaceBuffer) = 0;
 #endif
-    virtual RefPtr<PixelBuffer> drawingBufferToPixelBuffer(FlipY) = 0;
+    virtual RefPtr<PixelBuffer> drawingBufferToPixelBuffer(FlipY, RefPtr<WebCore::PixelBuffer>&&) = 0;
```

Comment:
I don't think these sort of APIs make sense to add, e.g. accepts pixelbuffer and returns pixelbuffer.

Currently polymorphic creation of PixelBuffer is not possible inside the API, so it creates a complication.
Same complication exists in the ImageBuffer case. See below.

The main API should likely be as it is today, and then the IPC implemetation should likely follow same pattern as

surfaceBufferToImageBuffer -> drawSurfaceBufferToImageBuffer

drawingBufferToPixelBuffer -> drawDrawingBufferToImageBuffer.